options/options.go

```diff
@@ -88,6 +88,8 @@ const (
 	FormatDefault = "default"
 	// FormatRaw specifies that results should be output in raw format.
 	FormatRaw = "raw"
+	// FormatStatement specifies that results should be output in an in-toto statement.
+	FormatStatement = "statement"
```

Comment:
discussion: any preference on `statement` vs `intoto` ?

Reply:
I also considered `attestation`.

Reply:
I think `intoto` would make it crystal clear what the output format would be.

Reply:
Let's do `FormatIntoto = "intoto"`, I'll leave a few other comments on names.